eclipse-che/che-devfile-registry
# chore: update version of che-code-devworkspace-handler lib

Signed-off-by: Valerii Svydenko <vsvydenk@redhat.com>



### What does this PR do?
Updates version of che-code-devworkspace-handler library to 1.64.0-dev-210b722, it's the version which is used in dashboard.

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/21543

### How to test this PR?

Build devfile registry and see that devworkspace-che-code-insiders.yaml files are generated

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
